playgrounds/skeleton-svelte/src/routes/components/tags-input/+page.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+	import { TagsInput } from '@skeletonlabs/skeleton-svelte';
+</script>
+
+<TagsInput>
+	<TagsInput.Label>Label</TagsInput.Label>
+	<TagsInput.Control>
+		<TagsInput.Input placeholder="Enter value..." class="w-full" />
```

Comment:
Note I've made three changes here:

1. Added a placeholder
2. Add `w-full`
3. Moved this above `<TagsInput.Context>`

This helps replicate the v3 style of display for the component. I don't think this *has* to be the default display, but we should show folks they can both replicate the v3 style, as well as use the "inline" tags ala Zag's defaults.

Maybe implement this as an alternative design example for folks that want to match the v3 styling?